### PR TITLE
Fix Editor e2e placeholder pixel rounding failure

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
Builds start to fail with 1 to 3 pixels difference when comparing expected position. 
Fixed by using a more relaxed condition.

## Motivation and Context
Fix build failures.

## How Has This Been Tested?
Tests pass.

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -364,7 +364,7 @@ gulp.task("server", factory.testServer({
 gulp.task("server-close", factory.testServerClose());
 gulp.task("test:webdriver_update", factory.webdriverUpdateSpecific({
     browsers: ["gecko=false"],
-    webdriverManagerArgs: ["--versions.chrome=" + (process.env.CHROME_VERSION || "latest")]
+    webdriverManagerArgs: ["--versions.chrome=84.0.4147.30"]
```
Comment (alex-deaconu) [inline gulpfile.js:367]: @ezequielc you can just use 84 here. Thanks.

Comment (ezequielc) [reply gulpfile.js:367]: Thanks!

Comment (ezequielc) [reply gulpfile.js]: @alex-deaconu Looks like it didn't work. I will revert to `84.0.4147.30`. Thanks

Comment (alex-deaconu) [reply gulpfile.js:367]: @ezequielc oops, sorry about that! I was certain that's how it works but you're right. Thanks.

**Comment by ezequielc** [comment]
As reference, this is the issue that forced us to set a fixed chrome driver version: https://bugs.chromium.org/p/chromedriver/issues/detail?id=3578. Thanks

**Comment by ezequielc** [comment]
@stulees fyi, build now should be fixed. Thanks

**Pull request merged**